DigitalLearningSolutions.Web/Views/LearningMenu/Section/_TutorialCard.cshtml

```diff
@@ -3,22 +3,25 @@
 <div class="nhsuk-card nhsuk-u-margin-bottom-3 learning-menu-card">
   <div class="nhsuk-card__content">
     <div class="nhsuk-grid-row">
-      <div class="@(Model.ShowLearnStatus ? "nhsuk-grid-column-three-quarters" : "nhsuk-grid-column-full")">
```

Comment:
When you integrate with the backend you probably need to change this again as we won't always have a recommended/optional tag to show. But fine to leave like this for now 🙂